public/javascripts/SVValidate/src/label/LabelControl.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Handles the hiding and showing of labels in the Google StreetView panorama.
+ * This is also called by the Keyboard class to deal with hiding the label
+ * via keyboard shortcuts.
+ * @returns {LabelControl}
+ * @constructor
+ */
+function LabelControl () {
+    var self = this;
+    var visible = true;
+    var labelControlButton = $("#label-control-button");
+
+    /**
+     * Logs interaction when the hide label button is clicked.
+     */
+    function clickHideLabel () {
+        svv.tracker.push("Click_HideLabel");
```

Comment:
If we are unhiding the label we should be logging "Click_UnhideLabel"

Reply:
Fixed